src/style.css

```diff
@@ -0,0 +1,57 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/AvenirLight.woff") format("woff");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/AvenirHeavy.woff") format("woff");
+  font-style: normal;
+  font-weight: 800;
+}
+
+.input {
+  width: 100%;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 800;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 63px;
+  color: #3d4e61;
+  height: 70px;
+  background: #fff url(../src/images/Search19.svg) 27px 50% no-repeat;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+}
+
+.input::placeholder {
+  font-weight: 300;
+}
+
+.input:hover {
+  outline: none;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  outline: none;
+  box-shadow: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.2);
+}
+
+.input-small {
+  font-size: 14px;
+  line-height: 19.12px;
```

Comment:
Don't apply decimal values, no need to copy all styles from mockup

Reply:
https://github.com/mate-academy/layout_search-bar-airbnb/pull/3210/files#r1161302562